src/relay/ir/pretty_printer.cc

```diff
@@ -181,9 +184,9 @@ class PrettyPrinter :
 
   Doc PrintAttrs(const Attrs& attrs, const Expr& op);
 
-  Doc Print(const NodeRef& node, bool meta = false) {
+  Doc Print(const NodeRef& node, bool meta = false, bool try_inline = false) {
```

Comment:
Make these two member of the language instead of passing them around.
If you pass them around you should change them. If they are constant save yourself the work.

Reply:
They are flags for the `Print` function and are changed by the callers in specific instances. They are only changed for one invocation of `Print` at a time and importantly not for recursive invocations. I think they make the most sense as flags on `Print` rather than as instance variables.